sceptre/scipool/config/develop/sc-product-ec2-linux-jumpcloud-notebook.yaml

```diff
@@ -99,4 +99,4 @@ sceptre_user_data:
     - Description: 'Update distro, python, apache, rstudio versions {{ range(1, 10000) | random }}'
       Info:
         LoadTemplateFromURL: 'https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/{{stack_group_config.service_catalog_library}}/v1.2.1/ec2/sc-ec2-linux-jumpcloud-notebook.yaml'
-      Name: 'v1.2.1'
+      Name: 'v1.2.4'
```

Comment:
there's a `v1.2.1` on the line above as well that needs to be updated

Reply:
Need new glasses...